main.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
```

Comment:
Use bits/stdc++.h for all required cpp headers in all files

Reply:
done